### PR TITLE
EVG-5185 use more correct query for stranded tasks, log host deco mor…

### PR DESCRIPTION
**Pull request by john-m-liu**
…e consistently

### CODE REVIEW

File: model/host/host.go [hunk 1]
```diff
@@ -999,7 +999,7 @@ func FindHostsSpawnedByBuild(buildID string) ([]Host, error) {
 
 func FindTerminatedHostsRunningTasks() ([]Host, error) {
 	hosts, err := Find(db.Query(bson.M{
-		StatusKey: bson.M{"$nin": evergreen.UphostStatus},
+		StatusKey: bson.M{"$nin": evergreen.RunningHostStatuses},
```
Comment (bsamek) [inline model/host/host.go]: Could this just be `StatusKey: evergreen.HostTerminated`?

File: service/distros.go [hunk 2]
```diff
@@ -116,13 +116,23 @@ func (uis *UIServer) modifyDistro(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if shouldDeco {
+		hosts, err := host.Find(host.ByDistroId(newDistro.Id))
+		if err != nil {
+			message := fmt.Sprintf("error finding hosts hosts: %s", err.Error())
```
Comment (bsamek) [inline service/distros.go]: nit error message, "hosts hosts"

File: model/host/host.go (hunk 1 above)
Comment (john-m-liu) [reply model/host/host.go]: yep, that's a lot better

File: service/distros.go (hunk 2 above)
Comment (john-m-liu) [reply service/distros.go]: fixed

**Pull request merged**